### ISSUE TITLE
Editing livereveal  in notebook metadata

### ISSUE THREAD

**Issue by btengels** [damianavila/RISE#172]
Trying to add the following snippit to the metadata

    "livereveal": {
        "theme": "sky",
        "transition": "none"
    },

But nothing changes. In fact, it doesn't matter what the 'theme' or 'transition' values are set to. Tested in Chrome and firefox. Pasted right before the "kernelspec" block as suggested in the readme.

btw, Damián this project is really the greatest. Bien hecho.

**Comment by bs1234** [comment]
I found a similar issue with transitions and width / height as well.
In case it helps you: when i shutdown my notebook and restarted the changes were in effect.
Might or might not be similar for you. Might or might not be something to do with caching.

**Comment by damianavila** [comment]
yep, caching could be an issue, try hard-reloading the page, disabing/cleaning cache or in incognito mode...

**Comment by parmentelat** [comment]
same issue here, it helps to know it's a matter of caching, although some deterministic method to get the new attributes to kick in would be helpful

regardless, this is an awesome tool, thanks a million for putting this together

**Comment by frederikph** [comment]
I am having a similar issue. Unfortunately, restarting the notebook and clearing cache doesn't work.

**Comment by jason-s** [comment]
I ran across the problem as well. There's a bug, which I figured out when I added some `console.log` statements in `main.js`.

This section of code:

```
config_section.load();

if(IPython.notebook.metadata.livereveal !== undefined){
    default_config = $.extend(true, default_config, IPython.notebook.metadata.livereveal);
}
var config = new configmod.ConfigWithDefaults(config_section, default_config);
```

the if statement is never true. It appears as though it executes before the metadata is actually loaded from the notebook.

Same with the `setup()` function, that runs before the notebook metadata is loaded.

Not sure when the right time is to grab the metadata and use them. See https://github.com/ipython/ipython/issues/7528
